sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpSender.cs

```diff
@@ -184,7 +192,7 @@ internal class AmqpSender : TransportSender
             options.MaxSizeInBytes ??= MaxMessageSize;
 
             Argument.AssertInRange(options.MaxSizeInBytes.Value, ServiceBusSender.MinimumBatchSizeLimit, MaxMessageSize.Value, nameof(options.MaxSizeInBytes));
-            return new AmqpMessageBatch(options);
+            return new AmqpMessageBatch(new AmqpMessageConverter(), options);
```

Comment:
Should we consider having a static property on AmqpMessageConverter that gives us an AmqpMessageConverter instance? I think we are only making the class non-static for testing, but there isn't actually any state that needs to be maintained in the instance.
/cc @jsquire

Reply:
Depends on the usage.  I believe we still intentionally pass an instance of the converter around in the infrastructure in a few places to enable testing.  If those places start to refer to the static property rather than accepting an instance via constructor injection, we're likely to have some test issues and, ultimately, a less testable surface.

Reply:
It's fine to still pass the instance in everywhere, but it would be more efficient if we could use a single static instance.

Reply:
Seems like it would be an easy perf win if we could hook everything up.